src/interfaces/swig/Evaluation.i

```diff
@@ -14,56 +14,16 @@
 /* Remove C Prefix */
 %rename(Evaluation) CEvaluation;
 %rename(EvaluationResult) CEvaluationResult;
-%rename(BinaryClassEvaluation) CBinaryClassEvaluation;
-%rename(ClusteringEvaluation) CClusteringEvaluation;
 %rename(ClusteringAccuracy) CClusteringAccuracy;
 %rename(ClusteringMutualInformation) CClusteringMutualInformation;
-%rename(ContingencyTableEvaluation) CContingencyTableEvaluation;
 %rename(MachineEvaluation) CMachineEvaluation;
-%rename(MulticlassAccuracy) CMulticlassAccuracy;
-%rename(MeanAbsoluteError) CMeanAbsoluteError;
-%rename(MeanSquaredError) CMeanSquaredError;
-%rename(MeanSquaredLogError) CMeanSquaredLogError;
-%rename(ROCEvaluation) CROCEvaluation;
-%rename(PRCEvaluation) CPRCEvaluation;
-%rename(AccuracyMeasure) CAccuracyMeasure;
-%rename(ErrorRateMeasure) CErrorRateMeasure;
-%rename(BALMeasure) CBALMeasure;
-%rename(WRACCMeasure) CWRACCMeasure;
-%rename(F1Measure) CF1Measure;
-%rename(CrossCorrelationMeasure) CCrossCorrelationMeasure;
-%rename(RecallMeasure) CRecallMeasure;
-%rename(PrecisionMeasure) CPrecisionMeasure;
-%rename(SpecificityMeasure) CSpecificityMeasure;
 %rename(SplittingStrategy) CSplittingStrategy;
-%rename(GradientCriterion) CGradientCriterion;
-%rename(GradientEvaluation) CGradientEvaluation;
-%rename(MulticlassOVREvaluation) CMulticlassOVREvaluation;
-%rename(CrossValidationResult) CCrossValidationResult;
-%rename(StructuredAccuracy) CStructuredAccuracy;
-%rename(DirectorContingencyTableEvaluation) CDirectorContingencyTableEvaluation;
-%rename(DifferentiableFunction) CDifferentiableFunction;
 
 /* Include Class Headers to make them visible from within the target language */
-%include <shogun/evaluation/EvaluationResult.h>
 %include <shogun/evaluation/Evaluation.h>
-%include <shogun/evaluation/BinaryClassEvaluation.h>
+%include <shogun/evaluation/EvaluationResult.h>
 %include <shogun/evaluation/ClusteringEvaluation.h>
 %include <shogun/evaluation/ClusteringAccuracy.h>
 %include <shogun/evaluation/ClusteringMutualInformation.h>
```

Comment:
@karlnapf can't remove these because of :
https://github.com/shogun-toolbox/shogun/blob/c948611b1e075b731d92c109d22abe79279b0a77/src/shogun/evaluation/ClusteringEvaluation.h#L36
not sure how to handle this?

Reply:
I remember I discussed this with someone 
Think we need to refactor this so that either the best map is computed automatically by the eval (once) or make the user do the call (eg using a “run(“compute_best_map”) and a runtime error in eval if the user hasn’t done that)

Reply:
hmm, i see, i will look into that!